src/ankh/cmd/ankh/ankh.go

```diff
@@ -0,0 +1,262 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/jawher/mow.cli"
+	"github.com/sirupsen/logrus"
+
+	"gopkg.in/yaml.v2"
+
+	"ankh"
+	"ankh/helm"
+	"ankh/kubectl"
+)
+
+var log = logrus.New()
+
+func main() {
+	app := cli.App("ankh", "Another Kubernetes Helper")
+	app.Spec = "[-v] [--ankhconfig] [--kubeconfig] [--datadir]"
+
+	var (
+		verbose    = app.BoolOpt("v verbose", false, "Verbose debug mode")
+		ankhconfig = app.StringOpt("ankhconfig", path.Join(os.Getenv("HOME"), ".ankh/config"), "The ankhconfig to use.")
+		kubeconfig = app.StringOpt("kubeconfig", "", "The kubeconfig to use when invoking kubectl.")
+		datadir    = app.StringOpt("datadir", path.Join(os.Getenv("HOME"), ".ankh"), "The data directory for ankh template history.")
+	)
+
+	formatter := logrus.TextFormatter{
+		DisableTimestamp: true,
+	}
+	log.Out = os.Stdout
+	log.Formatter = &formatter
+
+	ctx := &ankh.ExecutionContext{}
+
+	app.Before = func() {
+		if *verbose {
+			log.Level = logrus.DebugLevel
+		} else {
+			log.Level = logrus.InfoLevel
+		}
+
+		ctx = &ankh.ExecutionContext{
+			ConfigPath: *ankhconfig,
+			DataDir:    *datadir,
+			KubeConfig: *kubeconfig,
+			Logger:     log,
+		}
+
+	}
+
+	app.Command("apply", "Deploy an ankh file to a kubernetes cluster", func(cmd *cli.Cmd) {
+		cmd.Spec = "[-f] [--dry-run]"
+
+		filename := cmd.StringOpt("f filename", "ankh.yaml", "Config file name")
+		dryRun := cmd.BoolOpt("dry-run", false, "Perform a dry-run and don't actually apply anything to a cluster")
+
+		cmd.Action = func() {
+			ankhConfig, err := ankh.GetAnkhConfig(ctx)
+			check(err)
+
+			rootAnkhFile, err := ankh.ParseAnkhFile(*filename)
+			check(err)
+
+			log.Infof("Using ankh file %v", *filename)
+
+			dependencies := rootAnkhFile.Dependencies
+			if len(rootAnkhFile.AdminDependencies) > 0 {
+				log.Infof("Found admin dependencies, processing those first...")
+				dependencies = append(rootAnkhFile.AdminDependencies, dependencies...)
+			}
+
+			for _, dep := range dependencies {
+				prev, err := os.Getwd()
+				check(err)
+
+				err = os.Chdir(dep)
+				check(err)
+
+				// Should this not be hard coded?
```

Comment:
I'm in favor of leaving it hard coded unless we have a strong reason to allow different names.